Simulator 2.x/README.md

```diff
@@ -3,6 +3,10 @@ Mit diesem Flow kann eine komplette virtuelle Umgebung für openWB bereitgestell
 Die Umgebung besteht aus einem Zähler (muss als "EVU" in der Hierarchie eingerichtet sein),
 einem PV-Modul, einem Batteriespeicher sowie drei Ladepunkten.
 
+Im Ordner "extended version" findet sich ein Ausbau des Simulators um komplexere Szenarien des Lastmangements nachzustellen bei denen auch EVU Kits verschachtelt werden können.
+
+Passend dazu findet sich dort auch eine Standalonconfig in dem eine entsprechende Struktur angelegt ist. Einzig die IP der Standalone ist im Simulator anschließend noch in Betrieb zu nehmen.
+
```

Comment:
```suggestion
```
Es macht keinen Sinn, für eine simple Konfiguration ein komplettes Backup bereitzustellen. Der größte Nachteil ist, dass der Softwarestand schnell veraltet ist und regelmäßig aktualisiert werden müsste.

Besser eine eigene `README.md` in dem neuen Verzeichnis anlegen und mit ein paar Screenshots die notwendigen Einstellungen erläutern.